src/rlx_overlay.erl

```diff
@@ -17,7 +17,10 @@ render(Release, State) ->
             erlang:error(?RLX_ERROR(Reason));
         OverlayVars ->
             Files = rlx_util:template_files(),
-            do_overlay(State, Release, Files, OverlayVars)
+            case do_overlay(State, Release, Files, OverlayVars) of
+		{error, _} = E -> erlang:error(E);
```

Comment:
Indenting issue with these lines.

Reply:
Weird, when I do "view file" the indenting is fine...

Reply:
it's because it uses tabs, and the tab width changes based on the rendering

Reply:
changed the indentation to spaces.